base/steer/FairRun.cxx

```diff
@@ -21,7 +21,11 @@
 #include "FairRuntimeDb.h"      // for FairRuntimeDb
 #include "FairTask.h"           // for FairTask
 
-#include <TFile.h>     // for TFile
+#include <TFile.h>         // for TFile
+#include <TGeoManager.h>   // for gGeoManager, TGeoManager
+#if ROOT_VERSION_CODE >= ROOT_VERSION(6, 18, 2) && ROOT_VERSION_CODE <= ROOT_VERSION(6, 22, 6)
```

Comment:
Can you please add `#include <RVersion.h>`, thx!

Reply:
Done.